src/components/telemetry_monitor/src/metric_wrapper.cc

```diff
@@ -68,6 +68,10 @@ void MetricWrapper::Clear() {
   }
 }
 
```

Comment:
Method can be const an return const value or even const reference

Reply:
@LuxoftAKutsan fixed in https://github.com/Ford-Luxoft/sdl_core/pull/30/commits/3f042c33cb3c723382c0e0e040bc84a0158b1e70